projects/driver_controls/inc/cruise.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+// Cruise control target speed module
+// Requires CAN to be initialized
+// Stores updates to target cruise speed, reads motor speed from the motor controllers
+// Set the source to motor controller to record the current speed when driving normally.
+// Set the source to stored value to start cruise with the current speed and offset it to change the
+// target speed.
+#include <stdbool.h>
+#include <stdint.h>
+#include "event_queue.h"
+#include "status.h"
+
+// How to much to increment/decrement in cm/s (m/s * 100)
+// Arbitrary default of ~1mph
+#define CRUISE_OFFSET_CMS 45
```

Comment:
Can you explain how the math for this works somewhere?

Reply:
That's just the conversion from MPH to cm/s